Meetings.md

```diff
@@ -4,6 +4,7 @@ Individual session links and recordings are given below, most recent meeting at
 
 | Session  | Date  | Topic  | Links  |
 |---|---|---|---|
+| Phase 3, Session 1 | Friday February 8, 2019 [(video)]() | Hacking | [Project List]() |
```

Comment:
Let's start a new table for phase 3, and then add 2 sessions in the future to show what's coming up. And let's leave the free-form hack sessions off, but let the club know what the plan is in Slack.

Reply:
Done!  --Greg